scripts/azureml-assets/azureml/assets/release/asset_publish.py

```diff
@@ -73,23 +74,30 @@ def test_files_preprocess(test_jobs, asset_ids: dict):
 
     if publish_list_dir:
         with open(publish_list_dir) as fp:
-            publish_list = yaml.load(fp, Loader=yaml.FullLoader)
+            config = yaml.load(fp, Loader=yaml.FullLoader)
+            publish_list = config['create']
+            if publish_list is None:
```

Comment:
This will work with both an empty and non-existent list.
```suggestion
            if not publish_list:
```